# Stefan Teleman <stefan.teleman@oracle.com>
# As we were saying. Here it is. Removing --enable-largefile when
# we explicitly said we wanted largefile.
# The various ./configure's in GCC keep playing with CFLAGS, CXXFLAGS
# or CPPFLAGS, and this unsets LARGEFILE_SOURCE. THe result is that some
# GCC libraries are built with LARGEFILE_SOURCE enabled, and some don't.
# Make sure everything is built with LARGEFILE_SOURCE turned on.
# Internal patch. Not submitted upstream yet.
--- configure	2014-07-26 08:18:04.000000000 -0700
+++ configure	2015-08-05 08:25:33.246800903 -0700
@@ -6405,7 +6405,7 @@
   new_enable_languages=,c,
 
   # If LTO is enabled, add the LTO front end.
-  extra_host_libiberty_configure_flags=
+  extra_host_libiberty_configure_flags=--enable-largefile
   if test "$enable_lto" = "yes" ; then
     case ,${enable_languages}, in
       *,lto,*) ;;
@@ -6413,7 +6413,7 @@
     esac
     if test "${build_lto_plugin}" = "yes" ; then
       configdirs="$configdirs lto-plugin"
-      extra_host_libiberty_configure_flags=--enable-shared
+      extra_host_libiberty_configure_flags='--enable-largefile --enable-shared'
     fi
   fi
 
